test-utils/cli/src/lib.rs

```diff
@@ -35,6 +35,35 @@ use std::{
 };
 use tokio::io::{AsyncBufReadExt, AsyncRead};
 
+/// Similar to [`crate::start_node`] spawns a dev node on port `45789`, but works in environments
+/// where the substarte binary is not accessible with `cargo_bin("substrate-node")`.
+///
+/// Helpful if you need a Substrate dev node running in the background of a project external to
+/// `substrate`.
+///
+/// The downside compared to using [`crate::start_node`] is that this method is blocking rather than
+/// returning a [`Child`]. Therefore, you may want to call this method inside a new thread.
+///
+/// # Example
+/// ```ignore
+/// // Spawn a dev node in the background.
+/// let _ = std::thread::spawn(move || {
+/// 	common::start_node_without_binary();
+/// });
+/// ```
+pub fn start_node_without_binary() -> Result<(), sc_service::error::Error> {
```

Comment:
```suggestion
pub fn start_node_inline() -> Result<(), sc_service::error::Error> {
```

It's a personal preference though